tests/test_bench.py

```
@@ -1509,3 +1509,146 @@ def test_judge_verdicts_use_boxing_round_corner_labels(self):
+    the template must render them as a table — present when scored, absent when
+    the verdict is unscored (synthetic empty-tie)."""
+
+    @staticmethod
+    def _scores(left_val: int, right_val: int) -> dict[str, int]:
+        from evals.judges.pairwise_judge import _CRITERIA
+
+        d = {f"left_{c}": left_val for c in _CRITERIA}
+        d.update({f"right_{c}": right_val for c in _CRITERIA})
+        return d
+
+    def _build(self, *, scored: bool):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_build`

(ANN202)